ambuild2/task.py

```diff
@@ -66,12 +64,6 @@ def __init__(self, pump, channel, vars):
     })
 
   def receiveTask(self, channel, message):
-    task_id = message['task_id']
-    task_type = message['task_type']
-    task_folder = message['task_folder']
-    if not task_folder:
-      task_folder = '.'
```

Comment:
What happened to this check here?

Why did the task_type variable get folded into an expression? (I generally split things like this out for easier debugging.)